src/tests/ftest/daos_test/suite.yaml

```diff
@@ -9,6 +9,7 @@ timeout: 600
 timeouts:
   test_daos_degraded_mode: 450
   test_daos_management: 110
+  test_daos_cat_recovery: 3000
```

Comment:
50 minutes for a PR test? This will greatly slow down CI

Reply:
> 50 minutes for a PR test? This will greatly slow down CI

There are several cycles system start and stop for each test cases, so the test takes relative longer time than others. Do you have some idea to speedup that? Thanks!

Reply:
@daltonbohning Checking new tests tagged with `pr` that have excessive run times in their YAML configuration would be a **great** addition to your [ftest tag linting PR](https://github.com/daos-stack/daos/pull/13642).
.